dashboard/app/assets/stylesheets/projects.scss

```diff
@@ -17,81 +17,3 @@
     }
   }
 }
-
-$projects-table-width: 970px;
-
-.projects-table-wrapper {
-  width: $projects-table-width;
-}
-
-.projects {
-  width: $projects-table-width;
-  table-layout: auto;
-  margin: 10px 0;
-
-  th {
-    padding: 10px;
-    background-color: $bkgnd_color;
-    color: $white;
-    text-align: left;
-
-    a, a:hover, a:link, a:visited {
-      color: $white !important;
-    }
-  }
-  tr {
-    vertical-align: top;
-  }
-  td {
-    padding: 10px;
-    border: 1px solid $lighter_gray;
-  }
-  .name {
-    font-size: 1.5em;
-  }
-
-  input {
-    width: 90%;
-  }
-}
-
-.level-buttons {
-  margin: 20px 5px;
-}
-
-.project-links {
-  .level_link,.level_link:link, .level_link:visited {
-    background-color: $purple;
-    border-color: $purple;
-    font-size: 14px;
-    padding: 5px 10px;
-    color: $white;
-    &:hover {
-      background-color: $level_current;
-      border-color: $level_current;
-      text-decoration: none;
-    }
-  }
-}
-
-.loading {
-  height: 200px;
-  background: image-url("spinner-big.gif") no-repeat center center;
-}
```

Comment:
This is probably the riskiest part of the change, because the word "loading" is common and this rule is not scoped in any way to the old projects page. However, I feel ok about it because I was able to convince myself that the following occurrences should still be working properly:
* https://github.com/code-dot-org/code-dot-org/blob/34b12536c3f547ffaef61711245363e64d69b08e/apps/src/applab/commands.js#L2268
* https://github.com/code-dot-org/code-dot-org/blob/7ce88d6ee964da290841eaa576b3113305ad4a29/apps/src/code-studio/components/progress/MiniView.jsx#L49
* https://github.com/code-dot-org/code-dot-org/blob/48a5dd9cbdd8a7e3f5cbf426c782d4b15242fa56/dashboard/app/views/levels/_scratch.html.haml#L2-L1
* https://github.com/code-dot-org/code-dot-org/blob/56070f7f945b362f607808f2d7dec28ae06b493f/dashboard/app/views/levels/_blockly.html.haml#L10